AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -67,6 +67,7 @@ up-load.io,uploadrar.net,uploadev.org,ayefiles.com,douploads.com,fflares.com,fil
 ba-computer.at,stoertebeker.com,inateck.com,biershop-bayern.de,nkd.com,4f.com.pl,alcatelmobile.com,altex.ro,berliner-zeitung.de,bonne-maman.com,gabor.de,hallhuber.com,longines.co.uk,longines.fr,nanu-nana.de,relaiscolis.com,rossignol.com,sia.aviation-civile.gouv.fr,sportyfeel.fi###notice-cookie-block
 !
 twitch.tv##div.consent-banner
+jimmygreen.com##.gdprModal
```

Comment:
It was fixed here https://github.com/AdguardTeam/AdguardFilters/commit/0477b3e265b6334e1ebeeb0f8b7c9ce56cbdf115, please remove rule